scripts/draft-blog-post.mjs

```diff
@@ -74,7 +84,7 @@ fs.writeFileSync(
   replaceVersions(
     [
       fs.readFileSync(introFile, "utf8").trim(),
-      "<!--truncate-->",
+      shouldPrintTruncate ? "<!--truncate-->" : "",
```

Comment:
Why not check if blog-post-intro.md contains `<!--truncate-->`

Reply:
Thanks! [17ba6d8](https://github.com/prettier/prettier/pull/12412/commits/17ba6d85293d8ebcd646e80516b96af17dc7ee49)